lib/ui/window/viewport_metrics.h

```diff
@@ -32,9 +32,9 @@ struct ViewportMetrics {
                   double p_physical_system_gesture_inset_bottom,
                   double p_physical_system_gesture_inset_left,
                   double p_physical_touch_slop,
-                  const std::vector<double> p_physical_display_features_bounds,
```

Comment:
This may happen in an animation no?  these vectors shouldn't be that big though.